### PR TITLE
TASK-44444 : change buybutton color when disabled

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue [hunk 1]
```diff
@@ -105,7 +105,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             :title="buttonTitle"
             :right="!$vuetify.rtl"
             absolute
-            class="white--text primary buyButton"
+            :class="displayBuyButton ? 'white--text btn btn-primary buyButton' : 'white--text  buyButton'"
```
Comment (ahamdi) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue]: Use a computed for this check

Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue]: done!

File: perk-store-webapps/src/main/webapp/css/main.less [hunk 2]
```diff
@@ -888,7 +889,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   .buyButton{
     margin-top: 10px;
   }
-
+  .disabledBuyButton{
+    background-color: #E0E0E0 !important;
```
Comment (ahamdi) [inline perk-store-webapps/src/main/webapp/css/main.less]: Could you use a variable here  ?

Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/css/main.less]: could not find it in variables list ! add new one?

File: perk-store-webapps/src/main/webapp/css/main.less [hunk 3]
```diff
@@ -888,7 +889,12 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+  }
+  .VuetifyApp .v-btn--disabled {
+    pointer-events: auto;
+  }
```
Comment (Rdidmohamedelhadi) [inline perk-store-webapps/src/main/webapp/css/main.less]: please change this, the CSS you add will apply to all the v-btn in the platform, you have to add these CSS in the parent class

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

File: perk-store-webapps/src/main/webapp/css/main.less (hunk 3 above)
Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/css/main.less]: it's just to make its title appears!

Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/css/main.less]: done!

File: perk-store-webapps/src/main/webapp/css/main.less (hunk 2 above)
Comment (ahamdi) [reply perk-store-webapps/src/main/webapp/css/main.less]: This color should havz been used in the product for disabled buttons, if not found we add a new one.

**Review by ahamdi — DISMISSED** [review]

File: perk-store-webapps/src/main/webapp/css/main.less (hunk 2 above)
Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/css/main.less]: it is used several times without variable. I'll add a new one.

**Review by Rdidmohamedelhadi — APPROVED** [review]
LGTM

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**